source/plugins/ruby/in_kubestate_hpa.rb

```diff
@@ -60,6 +60,7 @@ def start
         @condition = ConditionVariable.new
         @mutex = Mutex.new
         @thread = Thread.new(&method(:run_periodic))
+        @@kubeStateHPATelemetryTimeTracker = DateTime.now.to_time.to_i
```

Comment:
same here .. this can be instance variable   and doesnt require to be class variable since this only used within this plugin.  This same comment applicable for other similar instances as well.